libs/community/langchain_community/embeddings/voyageai.py

```diff
@@ -1,63 +1,20 @@
 from __future__ import annotations
 
-import json
 import logging
 from typing import (
     Any,
-    Callable,
     Dict,
     List,
     Optional,
-    Tuple,
-    Union,
-    cast,
 )
 
-import requests
 from langchain_core.embeddings import Embeddings
-from langchain_core.pydantic_v1 import BaseModel, Extra, SecretStr, root_validator
-from langchain_core.utils import convert_to_secret_str, get_from_dict_or_env
-from tenacity import (
-    before_sleep_log,
-    retry,
-    stop_after_attempt,
-    wait_exponential,
-)
+from langchain_core.pydantic_v1 import BaseModel, Extra, root_validator
+from langchain_core.utils import get_from_dict_or_env
 
 logger = logging.getLogger(__name__)
 
 
-def _create_retry_decorator(embeddings: VoyageEmbeddings) -> Callable[[Any], Any]:
```

Comment:
for backwards compatibility b/n old versions of langchain and new versions of community we should keep these variables around
```python
_check_response=None
_create_retry_decorator=None
embed_with_retry=None
```